data/shopify_liquid/deprecated_filters.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Only the content from `deprecated_filters.yml` as JSON, so that we can read the content with the existing code in `SourceIndex#load_file`

Reply:
🚀 Nice!